feature-libs/cart/quick-order/components/quick-order/form/quick-order-form.component.html

```diff
@@ -1,17 +1,24 @@
-<form
-  (ngSubmit)="search($event)"
-  [formGroup]="form"
-  class="quick-order-form-container"
->
+<form [formGroup]="form" class="quick-order-form-container">
   <div class="quick-order-form-input">
     <input
+      (blur)="onBlur($event.relatedTarget)"
+      (keydown.arrowdown)="focusNextChild()"
+      (keydown.arrowup)="focusPreviousChild()"
+      (keydown.enter)="addProduct($event)"
+      (keydown.escape)="clear($event)"
       [attr.aria-label]="'common.search' | cxTranslate"
       class="form-control"
       formControlName="product"
       placeholder="{{ 'quickOrderForm.placeholder' | cxTranslate }}"
       type="text"
     />
 
+    <cx-icon
+      (click)="clear($event)"
+      [type]="iconTypes.RESET"
+      class="reset"
+      tabindex="-1"
+    ></cx-icon>
     <cx-icon [type]="iconTypes.SEARCH" class="search" tabindex="-1"></cx-icon>
```

Comment:
two cx-icons with two different formatting, can we format the second one same as the first one? 
I mean 
<cx-icon
 prop
 prop
 ></cx-icon>
for the sake of the readability

Reply:
It is not possible to change it as prettier is formating it like that, in the first cx-icon there is max line length exceed so that is why it is wrapping in different shape.